### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by AchillesDougalis**
Added a README.md file for every sub folder in each module.

### CODE REVIEW

File: implementation/ePO_core/owl_ontology/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# owl ontology
+
+**Description**:
+
+
+The *owl_ontology* folder contains the ePO_core ontology files, as well as the restriction files. The files are available in both [RDF](https://www.w3.org/RDF/) and [Turtle](https://www.w3.org/TR/turtle/) formats.
```
Comment (cristianvasquez) [inline implementation/ePO_core/owl_ontology/README.md]: 'The files are available in both [XML/RDF](https://www.w3.org/TR/rdf-syntax-grammar) and [Turtle/RDF](https://www.w3.org/TR/turtle)'

**Review by cristianvasquez — CHANGES_REQUESTED** [review]
Looks good, but there are some links to change in the 'owl ontology' markdown files. Please adjust as necessary

Explanation: RDF is the model we are using to represent things, and we use XML or turtle to represent those things. 
I could come also with an RDF graph drawed by hand that conveys the same information, the Idea  is to have the choice, always leaving the possibility of trasforming the RDF representation into turtle and then to JsonLD, etc, without loosing information.

File: implementation/ePO_core/owl_ontology/README.md (hunk 1 above)
Comment (AchillesDougalis) [reply implementation/ePO_core/owl_ontology/README.md]: I see, thank you. I implemented the changes. I ll close this request and make a new one.

**Review by cristianvasquez — COMMENTED** [review]
looks ok to me!

**Comment by cristianvasquez** [comment]
Hi, @AchillesDougalis you accidentally closed the pull request. I can only merge open pull requests

**Comment by AchillesDougalis** [comment]
Hi, yes indeed because  created a new one
https://github.com/OP-TED/ePO/pull/564

**Pull request closed without merging**